src/debug_util.c

```diff
@@ -1,11 +1,16 @@
 
+/* CygWin does not have execinfo.h... */
+#ifndef __CYGWIN__
```

Comment:
i think this should be '#ifdef __GLIBC__` instead, because only glibc provides this non-standard header. it would also break on e.g. linux distributions with musl libc, such as alpine.

Reply:
also, rahter than hardcoding assumptions about environments, a configure test would be the right thing to do. "checking whether execinfo.h provides backtrace()... no"

Reply:
I'm very much in agreement with this. I took out all the checks in autoconf because they just weren't used in the codebase. Instead there is just guessing based on compiler/platform. In most cases, these should be config tests, conducted by both autoconf and cmake.